webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPEntityMapper.scala

```diff
@@ -602,7 +602,12 @@ class LDAPEntityMapper(
         id                   <- e.required(A_ACTIVE_TECHNIQUE_UUID)
         refTechniqueUuid     <- e.required(A_TECHNIQUE_UUID).map(x => TechniqueName(x))
         isEnabled             = e.getAsBoolean(A_IS_ENABLED).getOrElse(false)
-        isSystem              = e.getAsBoolean(A_IS_SYSTEM).getOrElse(false)
+        policyTypes        = e(A_POLICY_TYPES) match {
+                                  case Some(json) => json.fromJson[PolicyTypes].getOrElse(PolicyTypes.rudderBase)
+                                  case None       =>
+                                    if (e.getAsBoolean(A_IS_SYSTEM).getOrElse(false)) PolicyTypes.rudderSystem
+                                    else PolicyTypes.rudderBase
+                                }
```

Comment:
For compat, try to read policy Type, and if not here is system. When writting, we only need to persist policy type. 
At some point, we will be able to remove isSystem.